dcm4che-core/src/main/java/org/dcm4che3/io/DicomInputStream.java

```diff
@@ -831,10 +832,16 @@ private void checkIsThis(DicomInputStream dis) {
             throw new IllegalArgumentException("dis != this");
     }
 
-    private void skipAttribute(String message) throws IOException {
-        LOG.warn(message,
-                 new Object[] { TagUtils.toString(tag), length, tagPos });
+    /**
+     * @param message the message to use in the warning log message
+     * @param methodName the name of the method that is skipping the attribute
+     * @throws IOException potentially thrown when performing the 'skip' operation
+     */
+    private void skipAttribute(String message, String methodName) throws IOException {
+        String tagAsString = TagUtils.toString(this.tag);
+        LOG.warn(message, tagAsString, length, tagPos, methodName);
         skipFully(length);
+        LOG.debug("Skipped {} bytes for {}", length, tagAsString);
```

Comment:
Okay, now it is already merged. but as also noted in https://github.com/dcm4che/dcm4che/pull/1184#discussion_r872455148 I think this debug log message does not provide any additioal value, as all the information is anyways already contained in the warn log message above.